### PR TITLE
Adding logo and icon

### PR DESCRIPTION
**Pull request by abashir236**
This PR:

- adds/changes/removes etc

### Testing

Description on how {APP-NAME} can be tested.

- [ ] fresh install works
- [ ] upgrade from previous version works

#### Other testing

Description of features to additionally test for {APP-NAME} installations.

- [ ] check reconciliation of existing resources after upgrading
- [ ] X still works after upgrade
- [ ] Y is installed correctly

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update Lastpass values if required.

### CODE REVIEW

File: helm/cluster-cloud-director/Chart.yaml [hunk 1]
```diff
@@ -13,6 +14,7 @@ keywords:
   - vcd
 annotations:
   application.giantswarm.io/team: "rocket"
+  ui.giantswarm.io/logo: https://s.giantswarm.io/app-icons/vmware-cloud-director/1/logo_light.png
```
Comment (kopiczko) [inline helm/cluster-cloud-director/Chart.yaml:17]: Gives 404?

Comment (abashir236) [reply helm/cluster-cloud-director/Chart.yaml:17]: The PR with team-rainbow has not been merged yet

Comment (jkremser) [reply helm/cluster-cloud-director/Chart.yaml:17]: it's 200 now

**Review by jkremser — APPROVED** [review]

**Review by marians — APPROVED** [review]
Thanks!

**Pull request merged**